app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -62,6 +62,14 @@ def rrd_status
     'unknown'
   end
 
+  # If a pending_ep was detected in the past, returns true
+  # Depends on offramp_reason in form JSON, which is inserted by pending_eps? method
+  def had_pending_eps?
+    return true if form.dig('rrd_metadata', 'offramp_reason') == 'pending_ep'
+
+    pending_eps?
```

Comment:
Are you sure we want to call `pending_eps?` at all in this method?

Reply:
The follow-up PR still has this call to `pending_eps?`. https://github.com/department-of-veterans-affairs/vets-api/pull/10531/files#diff-b39fb435faa12348b70b1e18972038160150f4c440c2959c4a09773d46f284dfL70

Let's chat in Slack huddle if you still think it's needed.